packages/lazy-images/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@automattic/jetpack-lazy-images",
+	"version": "1.0.0",
+	"description": "Lazy Image Loading.",
+	"main": "dist/lazy-images.js",
+	"sideEffects": true,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/jetpack.git",
+		"directory": "packages/lazy-images"
+	},
+	"private": true,
+	"author": "Automattic Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"homepage": "https://github.com/Automattic/jetpack",
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prebuild": "yarn run clean",
+		"build": "calypso-build lazy-images='./src/js/lazy-images.js' intersectionobserver-polyfill='./src/js/intersectionobserver-polyfill.js'"
+	},
+	"dependencies": {
+		"intersection-observer": "^0.11.0"
```

Comment:
You added this as a dependency, which is good. But you're still using the local copy on line 22 and in src/lazy-images.php, which seems to defeat the purpose.

Reply:
Good spot! I was hoping to use the npm but then changed my mind, but forgot to remove this.

Reply:
Ended up with a solution that actually uses the file from the npm package :tada: